AutomatedTesting/Gem/PythonTests/prefab/Prefab_Operations.py

```diff
@@ -0,0 +1,190 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+from collections import Counter
+from collections import deque
+
+from PySide2 import QtWidgets
+
+from azlmbr.entity import EntityId
+from azlmbr.math import Vector3
+from editor_python_test_tools.utils import Report
+
+import azlmbr.bus as bus
+import azlmbr.editor as editor
+import azlmbr.prefab as prefab
+import editor_python_test_tools.pyside_utils as pyside_utils
+import prefab.Prefab_Test_Utils as prefab_test_utils
+
+
+class OperationResults:
+    create_entity_succeeded = (
+        "'CreateNewEntity' passed",
+        "'CreateNewEntity' failed")
+
+    create_prefab_succeeded = (
+        "'CreatePrefab' passed",
+        "'CreatePrefab' failed")
+
+    instantiate_prefab_succeeded = (
+        "'InstantiatePrefab' passed",
+        "'InstantiatePrefab' failed")
+
+    entity_has_expected_children_count = (
+        "target entity owns exactly given count of children",
+        "target entity does *not* owns given count of children")
+
+    delete_prefab_succeeded = (
+        "'DeleteEntitiesAndAllDescendantsInInstance' passed",
+        "'DeleteEntitiesAndAllDescendantsInInstance' failed")
+
+    delete_prefab_entities_and_descendants_succeeded = (
+        "Entities and all the descendants in target prefab are deleted",
+        "Entities and all the descendants in target prefab are *not* deleted")
+
+    reparent_prefab_no_cyclical_dependency_detected = (
+        "Cyclical dependency not detected while reparenting prefab",
+        "Cyclical dependency detected while reparenting prefab")
+
+    reparent_prefab_new_child_with_reparented_prefab_name_added = (
+        "An entity with reparented prefab name becomes a child of target parent entity",
+        "No entity with reparented prefab name become a child of target parent entity")
+
+    reparent_prefab_reparented_prefab_under_parent = (
+        "Prefab reparented is under the expected parent entity",
+        "Prefab reparented is *not* under the expected parent entity")
```

Comment:
If possible, use the same line and align the success with failures like happens with the rest of the tests

Reply:
I don't think that's a good idea since every time when you try to add new results it will have to adjust almost all the previous ones. Also, not all the tests have this kind of convention right now.

Reply:
Right now I just declare the tuples inside the function that uses them.